Dockerfile

```diff
@@ -77,6 +77,7 @@ RUN apt-get update && \
         tzdata \
         libreadline8 \
         tini && \
+    rm -rf /var/lib/apt/lists/* && \
```

Comment:
There is a comment and haldolint ignore up on 58 that probably needs to be resolved

Reply:
oops, didn't notice DL3009, it's interesting to ignore that though